docker-compose.yml

```diff
@@ -120,6 +121,10 @@ services:
   snuba-sessions-consumer:
     << : *snuba_defaults
     command: consumer --storage sessions_raw --auto-offset-reset=latest --max-batch-time-ms 750
+  # Kafka consumer responsible for feeding transactions data into Clickhouse
+  snuba-transactions-consumer:
+    << : *snuba_defaults
+    command: consumer --storage transactions --auto-offset-reset=latest --max-batch-time-ms 750
```

Comment:
I am not sure what the reasoning was to use latest as auto offset reset, but since it is used for all the other consumers, i guess this should be connnsistent